src/ui/button/styles/button.js

```diff
@@ -72,4 +72,74 @@ export const buttonStyle = `
         height: 100%;
         text-align: left;
     }
+    .${ CLASS.SHAPE }-${BUTTON_SHAPE.PILL}  {
+        border-radius: 50px;
+    }
+    .${'PAYPAL-SPINNER'}{
+        display:block;
+    }
+    .${'PAYPAL-SPINNER'}.${'loader'}{
```

Comment:
doesn't the component support styling props ?

Reply:
removed the style